src/oaklib/converters/obo_graph_to_fhir_converter.py

```diff
@@ -290,3 +306,132 @@ def _convert_meta(self, source: Node, concept: Concept):
                     value=synonym.val,
                 )
             )
+
+    @staticmethod
+    def write_concept_maps(obograph_path: str, outdir: str = None, code_system_id: str = None, verbose=True):
+        """"From an Obograph JSON, convert to SSSOM, then convert to 1+ ConceptMap JSON"""
+        # Vars
+        outdir = outdir or obograph_path
+        outdir = outdir if os.path.isdir(outdir) else os.path.dirname(outdir)
+        # todo: could be more combinations like - or _ obographs
+        code_system_id = code_system_id or os.path.basename(obograph_path)\
+            .replace(".obographs", "").replace(".obograph", "").replace(".json", "")
+        outpath_sssom = os.path.join(outdir, f"{code_system_id}.sssom.tsv")
+
+        # TODO obograph -> sssom
+        #  - needs path to be passed to dump
+        #  - before i continue with this, check how owl-on-fhir does. does it use OAK CLI or Python API?
+        #   ...if CLI, then I need to somehow get the CLI to pass the path down here. maybe not difficult
+        #   ...or maybe i should just do that regardless
+        #   ...how did I call the CLI again? maybe i ned to check my notes or save better notes...
+
+        # todo: is there a way to do this via Python API? would be better
+        # todo: -m metadata.sssom.yml: can I create this file on the fly and pass?
+        command_str = f'sssom parse {obograph_path} -I obographs-json -o {outpath_sssom}'
+        print('Converting: Obographs -> SSSSOM')
+        result = subprocess.run(command_str.split(), capture_output=True, text=True)
+        stderr, stdout = result.stderr, result.stdout
+        if verbose:
+            print(stdout)
+            print(stderr, file=sys.stderr)
```

Comment:
## GO (& other foreseeable) missing namespace issues
@cmungall I used GO as my first test. Incidentally I think you are a GO expert. Can you help me think of a solution to this problem? GO (and perhaps other ontologies as well) may not have URIs for some xrefs. This results in the xrefs not being able to be rendered in SSSOM.

Example:
`WARNING:root:CURIE appeared where there should be a URI, and could not be standardized: Reactome:R-HSA-5668414`

<details><summary>Full conversion log</summary>
<p>

```
%]Converting: Obographs -> SSSSOM


WARNING:root:CURIE appeared where there should be a URI, and could not be standardized: Wikipedia:Cytoplasm
WARNING:root:CURIE appeared where there should be a URI, and could not be standardized: Reactome:R-HSA-5668414
WARNING:root:CURIE appeared where there should be a URI, and could not be standardized: Reactome:R-HSA-8868783
WARNING:root:CURIE appeared where there should be a URI, and could not be standardized: Reactome:R-HSA-1483089
WARNING:root:CURIE appeared where there should be a URI, and could not be standardized: EC:2.-.-.-
WARNING:root:CURIE appeared where there should be a URI, and could not be standardized: Reactome:R-HSA-1483186
WARNING:root:CURIE appeared where there should be a URI, and could not be standardized: Wikipedia:Intracellular
WARNING:root:CURIE appeared where there should be a URI, and could not be standardized: NIF_Subcellular:sao414196390
WARNING:root:CURIE appeared where there should be a URI, and could not be standardized: NIF_Subcellular:sao1539965131
WARNING:root:CURIE appeared where there should be a URI, and could not be standardized: Wikipedia:Organelle
WARNING:root:CURIE appeared where there should be a URI, and could not be standardized: Wikipedia:Photosynthesis
WARNING:root:CURIE appeared where there should be a URI, and could not be standardized: Wikipedia:Cell_cortex
WARNING:root:CURIE appeared where there should be a URI, and could not be standardized: Wikipedia:Biological_membrane
WARNING:root:CURIE appeared where there should be a URI, and could not be standardized: NIF_Subcellular:sao830981606
WARNING:root:CURIE appeared where there should be a URI, and could not be standardized: NIF_Subcellular:sao1337158144
WARNING:root:CURIE appeared where there should be a URI, and could not be standardized: NIF_Subcellular:sao1687101204
WARNING:root:CURIE appeared where there should be a URI, and could not be standardized: Wikipedia:Metabolism
WARNING:root:CURIE appeared where there should be a URI, and could not be standardized: Wikipedia:Biological_process
WARNING:root:CURIE appeared where there should be a URI, and could not be standardized: Wikipedia:Enzyme
WARNING:root:CURIE appeared where there should be a URI, and could not be standardized: Wikipedia:Endomembrane_system
WARNING:root:CURIE appeared where there should be a URI, and could not be standardized: Wikipedia:Vacuole
WARNING:root:CURIE appeared where there should be a URI, and could not be standardized: Reactome:R-HSA-6788855
WARNING:root:CURIE appeared where there should be a URI, and could not be standardized: Reactome:R-HSA-6788867
WARNING:root:CURIE appeared where there should be a URI, and could not be standardized: Wikipedia:Thylakoid
WARNING:root:CURIE appeared where there should be a URI, and could not be standardized: Wikipedia:Phosphorylation
WARNING:root:CURIE appeared where there should be a URI, and could not be standardized: Wikipedia:Nuclear_envelope
WARNING:root:CURIE appeared where there should be a URI, and could not be standardized: Wikipedia:Cell_nucleus
WARNING:root:CURIE appeared where there should be a URI, and could not be standardized: NIF_Subcellular:sao1702920020
WARNING:root:CURIE appeared where there should be a URI, and could not be standardized: Wikipedia:Cell_membrane
WARNING:root:CURIE appeared where there should be a URI, and could not be standardized: NIF_Subcellular:sao1663586795
WARNING:root:CURIE appeared where there should be a URI, and could not be standardized: EC:2.7.-.-

```

</p>
</details> 

This problem will be solved I suppose with a `metadata.yml`. But I don't know what URIs to use. And I do want this tool to be flexible enough to hopefully not require everyone wanting to convert OWL to FHIR to need to make these `metadata.yml` files. Should I just create artificial pURLs on the spot for each unrecognized namespace, then dynamically construct a `metadata.yml`?

Reply:
@cmungall You mentioned splitting this into a new PR, so this has for now been removed.